docs/source/user-guide/guide_api.rst

```diff
@@ -4,28 +4,17 @@ API Examples
 When visiting a dataset page, you will see dataset specific API examples
 for that dataset that do not require authentication.
 
-This guide will show additional options for more advanced usage.
-
-Authentication
---------------
-
-Drupal uses Basic Authentication, this is a method for an HTTP user agent (e.g., a web browser)
-to provide a username and password when making a request.
-
-When employing Basic Authentication, users include an encoded string in the Authorization
-header of each request they make. The string is used by the request's recipient to verify
-users' identity and rights to access a resource.
-
-  -  Key = Authorization
-  -  Value = Basic + base 64 encoding of a user ID and password separated by a colon
+This guide will provide examples that do and do not require authentication as well as
+additional options for more advanced usage.
 
 Identifiers
 -----------
 
 Every dataset has an identifier, a.k.a. UUID. The dataset identifier is used in the dataset URL (/dataset/datasetID),
 and does not change when edits are made to the dataset. So if you are creating an automated
 script it is better to use APIs that utilize the dataset ID and the index of the distribution.
-The index for the first distribution is 0, the index for the second distribution is 1, etc.
+The index for the first distribution is 0, the index for the second distribution is 1, etc. In most cases
```

Comment:
Suggest replacing the last two sentences with:

Commonly datasets have a single distribution, in this case, the index will always be 0. For datasets that have several distributions, the index corresponds to the order in which they were entered. The first distribution would be 0, the second would be 1, and so on.

Reply:
I like this! Adjusting -